### DISCUSSION TITLE
你们官网文档，用户指南里面竟然没有用户指南，只有个移植。

### DISCUSSION THREAD

**Discussion by zouzhe1** [halo-dev/halo#2328] [General]
### 检查清单

- [X] 已经在 [GitHub Issues](https://github.com/halo-dev/halo/issues) 中搜索过该问题
- [X] 已经在[论坛](https://bbs.halo.run/)中搜索过该问题
- [X] 已经查阅过[文档](https://docs.halo.run/)

### 是什么版本出现了此问题？

1

### 使用的什么数据库？

H2

### 使用的哪种方式部署？

Docker

### 在线站点地址

https://docs.halo.run/user-guide

### 发生了什么？

https://docs.halo.run/user-guide
你们官网文档，用户指南里面竟然没有用户指南，只有个移植。
没有网站用户指南操作讲解，没有后台各功能介绍。

**Comment by zouzhe1** [comment]
我甚至连后台登录地址说明都没有找到！！

**Comment by onlytl** [comment]
<img width="1913" alt="image" src="https://user-images.githubusercontent.com/43626236/185270658-d563755f-f247-42f8-8e8b-213fa9f45ce0.png">

是啊 ，什么都没有

**Reply by zouzhe1** [reply]
你没懂我意思，我说你们分类分错了，这些操作的事情，都应该放在用户指南下面。你们还应该有个页面专门写，默认地址，默认信息是啥啥啥。而不是把信息整到配置过程中，这样不清晰。我这是建议！愿意吸取建议，才会变得更好。

**Reply by JohnNiang** [reply]
@zouzhe1 很好的建议，关于“如何使用 Halo” 的文档确实有缺失，目前只能靠 Haloer 自行摸索才行。等我们稳定了 Halo 2.0 开始着手这方面的补充。

**Reply by ruibaby** [reply]
> 你没懂我意思，我说你们分类分错了，这些操作的事情，都应该放在用户指南下面。你们还应该有个页面专门写，默认地址，默认信息是啥啥啥。而不是把信息整到配置过程中，这样不清晰。我这是建议！愿意吸取建议，才会变得更好。

你应该说的是没有使用文档，但这个配置文档是专门用于讲解配置文件的，和你说的后台使用没有任何关系。另外，我从你的 issue 原文没有看出任何建议的意思。如果你想参与开源软件贡献，推荐阅读 https://github.com/LinuxSuRen/open-source-best-practice https://opensource.guide/zh-hans/
